modules/react/popup/stories/stories.tsx

```diff
@@ -15,67 +15,53 @@ import {
 import README from '../README.md';
 
 export default {
-  title: 'Components/Popups/Popup/React',
+  title: 'Components/Popups/Popup/React/Old',
```

Comment:
Can we just remove these? Again - I don't think we should document the old way of doing it.